js/data/locale/currency.json

```diff
@@ -42,32 +42,32 @@
     "CHF": {
         "name": "Swiss Franc",
         "decimals": 2,
-        "sign": "Fr"
+        "sign": "CHF"
```

Comment:
The symbols are in the "symbol-alt-narrow" property of the cldr json files.

Reply:
I'll update the extractor code then regenerate files.